types/feathersjs__feathers/tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-    "extends": "dtslint/dt.json"
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "file-name-casing": false
```

Comment:
Why did this have to be turned off?

Reply:
Because the file `feathersjs__feathers-tests.ts` wasn't following it, and so it fired off and failed linting. 

I didn't add the file. But the instructions said to test to ensure that lint rules were followed.